### PR TITLE
Swapping quick start

### PR DESCRIPTION
**Pull request by CumpsD**
Provide the swapping quick start flow as well as some extra info on why an broker api provider can be useful

### CODE REVIEW

File: pages/swapping/integrations/quick-start.mdx [hunk 1]
```diff
@@ -10,4 +10,60 @@ import { Callout } from "@/components";
 
 # Quick Start
 
-Coming soon.
+To quickly get started on Chainflip, there are multiple scenarios possible.
```
Comment (fernandezdavid) [inline pages/swapping/integrations/quick-start.mdx:13]: there are only 2, and could argue is just a test and prod scenarios

File: pages/swapping/integrations/quick-start.mdx [hunk 2]
```diff
@@ -10,4 +10,60 @@ import { Callout } from "@/components";
...
+Now that you have a running Broker API, you are ready to use our [Javascript SDK](javascript-sdk/introduction):
+
+* [Point to your broker and set a commission](javascript-sdk/quick-start#set-up)
+* [Request a deposit address](javascript-sdk/swap-assets/request-deposit-address)
+
+### 5. Swap!
+
+After having opened a deposit channel and having requested a deposit address, it is time to send funds to the deposit address and [watch the Chainflip magic happen!](../../concepts/swaps-amm/native-swap-flow)
+
+## Quick Scenario
```
Comment (fernandezdavid) [inline pages/swapping/integrations/quick-start.mdx:51]: I'd move this up and start with the "Quick scenario", that I'd also rename to "Development-only scenario" or something like that

File: pages/swapping/integrations/quick-start.mdx [hunk 3]
```diff
@@ -10,4 +10,60 @@ import { Callout } from "@/components";
...
+<Callout type="warning">
+The quick scenario is useful for a development environment. **It is not advised for production**!
+</Callout>
+
+A faster way to start developing is by swapping via the main Chainflip Broker. This **prevents you from earning any fees** and is subject to imposed rate limits.
+
+
+### 1. Use the JS SDK & open a deposit channel
+
+Now that you have a running Broker API, you are ready to use our [Javascript SDK](javascript-sdk/introduction):
```
Comment (fernandezdavid) [inline pages/swapping/integrations/quick-start.mdx:62]: I think this line is confusing, I'd just say something on the lines "Use the JS SDK straight with default config to make use of the Chainflip Broker"

File: pages/swapping/integrations/quick-start.mdx [hunk 4]
```diff
@@ -10,4 +10,60 @@ import { Callout } from "@/components";
...
+
+* [Set up the SDK with the default options](javascript-sdk/quick-start#set-up)
+* [Request a deposit address](javascript-sdk/swap-assets/request-deposit-address)
+
+### 2. Swap!
+
+After having opened a deposit channel and having requested a deposit address, it is time to send funds to the deposit address and [watch the Chainflip magic happen!](../../concepts/swaps-amm/native-swap-flow)
```
Comment (fernandezdavid) [inline pages/swapping/integrations/quick-start.mdx:69]: ..."By opening a deposit channel, the protocol generates a deposit address with an expiration time"...

File: pages/swapping/integrations/quick-start.mdx [hunk 5]
```diff
@@ -10,4 +10,60 @@ import { Callout } from "@/components";
 
 # Quick Start
 
-Coming soon.
+To quickly get started on Chainflip, there are multiple scenarios possible.
+
+## Ideal Scenario
+
+### 1. A little bit of background
```
Comment (fernandezdavid) [inline pages/swapping/integrations/quick-start.mdx:17]: "Understanding the protocol" as title

File: pages/swapping/integrations/quick-start.mdx [hunk 6]
```diff
@@ -10,4 +10,60 @@ import { Callout } from "@/components";
...
+* [Introduction on how swapping works](../how-swapping-works)
+* [The power of deposit channels](../../concepts/swaps-amm/deposit-channels-and-brokers)
+* [The role of the broker, and how to earn fees](running-a-broker/introduction)
+
+### 2. Prepare your Chainflip Account
+
+The first practical step to get started is to get your Chainflip Account up and running. In order to do this you need to:
+
+* [Create a Chainflip Account](chainflip-account/creating)
+* [Fund it with some FLIP](chainflip-account/funding) (10 FLIP)
```
Comment (fernandezdavid) [inline pages/swapping/integrations/quick-start.mdx:30]: Fund it with FLIP to pay for network gas

File: pages/swapping/integrations/quick-start.mdx [hunk 7]
```diff
@@ -10,4 +10,60 @@ import { Callout } from "@/components";
...
+
+### 1. A little bit of background
+
+To get you in the best position to work with Chainflip it is useful to have a little bit of knowledge on how the system works. Take a look at the following to understand the moving parts:
+
+* [Introduction on how swapping works](../how-swapping-works)
+* [The power of deposit channels](../../concepts/swaps-amm/deposit-channels-and-brokers)
+* [The role of the broker, and how to earn fees](running-a-broker/introduction)
+
+### 2. Prepare your Chainflip Account
```
Comment (fernandezdavid) [inline pages/swapping/integrations/quick-start.mdx:25]: "Creating your Broker account"

File: pages/swapping/integrations/quick-start.mdx [hunk 8]
```diff
@@ -10,4 +10,60 @@ import { Callout } from "@/components";
...
+
+The first practical step to get started is to get your Chainflip Account up and running. In order to do this you need to:
+
+* [Create a Chainflip Account](chainflip-account/creating)
+* [Fund it with some FLIP](chainflip-account/funding) (10 FLIP)
+* [Register your Account as a Broker](chainflip-account/broker)
+
+### 3. Setup a Broker
```
Comment (fernandezdavid) [inline pages/swapping/integrations/quick-start.mdx:33]: Running the Broker API

File: pages/swapping/integrations/quick-start.mdx [hunk 9]
```diff
@@ -10,4 +10,60 @@ import { Callout } from "@/components";
...
+
+Once you have your Broker account, it's time to get the Broker API running in order to communicate with the Chainflip network. There are two main ways to do this:
+
+* [Operate a Node and Broker API yourself](running-a-broker/broker-api)
+* [Utilize a Broker API Provider](running-a-broker/affiliate-brokers)
+
+### 4. Use the JS SDK & open a deposit channel
+
+Now that you have a running Broker API, you are ready to use our [Javascript SDK](javascript-sdk/introduction):
```
Comment (fernandezdavid) [inline pages/swapping/integrations/quick-start.mdx:42]: ..."you initialise the SDK with your Broker account ID"...

File: pages/swapping/integrations/quick-start.mdx [hunk 10]
```diff
@@ -10,4 +10,60 @@ import { Callout } from "@/components";
...
+* [Register your Account as a Broker](chainflip-account/broker)
+
+### 3. Setup a Broker
+
+Once you have your Broker account, it's time to get the Broker API running in order to communicate with the Chainflip network. There are two main ways to do this:
+
+* [Operate a Node and Broker API yourself](running-a-broker/broker-api)
+* [Utilize a Broker API Provider](running-a-broker/affiliate-brokers)
+
+### 4. Use the JS SDK & open a deposit channel
```
Comment (fernandezdavid) [inline pages/swapping/integrations/quick-start.mdx:40]: Using the JS SDK to open deposit channels

File: pages/swapping/integrations/quick-start.mdx [hunk 11]
```diff
@@ -10,4 +10,60 @@ import { Callout } from "@/components";
...
+
+Now that you have a running Broker API, you are ready to use our [Javascript SDK](javascript-sdk/introduction):
+
+* [Point to your broker and set a commission](javascript-sdk/quick-start#set-up)
+* [Request a deposit address](javascript-sdk/swap-assets/request-deposit-address)
+
+### 5. Swap!
```
Comment (fernandezdavid) [inline pages/swapping/integrations/quick-start.mdx:47]: Sending funds to swap

File: pages/swapping/integrations/quick-start.mdx [hunk 12]
```diff
@@ -10,4 +10,60 @@ import { Callout } from "@/components";
...
+
+
+### 1. Use the JS SDK & open a deposit channel
+
+Now that you have a running Broker API, you are ready to use our [Javascript SDK](javascript-sdk/introduction):
+
+* [Set up the SDK with the default options](javascript-sdk/quick-start#set-up)
+* [Request a deposit address](javascript-sdk/swap-assets/request-deposit-address)
+
+### 2. Swap!
```
Comment (fernandezdavid) [inline pages/swapping/integrations/quick-start.mdx:67]: Sending funds to swap

File: pages/swapping/integrations/quick-start.mdx (hunk 1 above)
Comment (CumpsD) [reply pages/swapping/integrations/quick-start.mdx:13]: I was thinking of "multiple" if you take into account not using the JS SDK and using straight RPC calls. But we can rename this to actually 1 scenario then if we call it dev and prod?

File: pages/swapping/integrations/quick-start.mdx (hunk 2 above)
Comment (CumpsD) [reply pages/swapping/integrations/quick-start.mdx:51]: Ok, I placed it at the bottom because people read from the top and then do the wrong thing in production :p

File: pages/swapping/integrations/quick-start.mdx (hunk 3 above)
Comment (CumpsD) [reply pages/swapping/integrations/quick-start.mdx:62]: Yes, this is a copy paste mistake....

File: pages/swapping/integrations/quick-start.mdx (hunk 9 above)
Comment (CumpsD) [reply pages/swapping/integrations/quick-start.mdx:42]: This is not actually true, you dont initalise the SDK with your broker account, you point it at the Broker API, which happens to hold the broker account keys

File: pages/swapping/integrations/quick-start.mdx (hunk 1 above)
Comment (fernandezdavid) [reply pages/swapping/integrations/quick-start.mdx:13]: I see, yeah I think that's pretty much the only 2 described in this page

File: pages/swapping/integrations/quick-start.mdx (hunk 2 above)
Comment (fernandezdavid) [reply pages/swapping/integrations/quick-start.mdx:51]: not if we add the dev and prod in the title lol

File: pages/swapping/integrations/quick-start.mdx (hunk 9 above)
Comment (fernandezdavid) [reply pages/swapping/integrations/quick-start.mdx:42]: you are right, let me rephrase "you initialise the SDK with your Broker service URL"

**Review by fernandezdavid — APPROVED** [review]
